alchemist/alchemist-cognitive-agents/src/test/kotlin/it/unibo/alchemist/test/TestFeelsTransmission.kt

```diff
@@ -0,0 +1,19 @@
+package it.unibo.alchemist.test
+
+import io.kotlintest.matchers.doubles.shouldBeGreaterThan
+import io.kotlintest.specs.StringSpec
+import it.unibo.alchemist.model.implementations.nodes.CognitivePedestrian2D
+import it.unibo.alchemist.model.interfaces.Position
+
+class TestFeelsTransmission<T, P : Position<P>> : StringSpec({
+
+    "danger layer affects cognitive pedestrians" {
+        val aggregateDangerWithoutLayer = loadYamlSimulation<T, P>("feels-transmission-without-layer.yml")
+                .nodes.map { it as CognitivePedestrian2D<T, *> }.sumByDouble { it.dangerBelief() }
```

Comment:
Looks a bit hard to read: maybe better if every chained call goes on its own line?